enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -341,185 +335,303 @@ risks and disruption when rolling out changes to their environments.
-strategy configuration for a MachineConfigPool, but may be meaningless for other resources). 
+#### Permissive Strategy
+A policy using the permissive strategy will always suggest a permissive window. A cluster lifecycle administrator
+may want to toggle a `ChangeManagementPolicy` from the `Restrictive` to `Permissive` strategy, and back again,
+as a means to implementing their own change management window mechanism.
 
-This strategy is motivated by the desire to support the separation of control-plane and worker-node updates both
-conceptually for users and in real technical terms. One way to do this for users who do not benefit from the
-`MaintenanceSchedule` strategy is to ask them to initiate, pause, and resume the rollout of material
-changes to their worker nodes. Contrast this with the fully self-managed state today, where worker-nodes
-(normally) begin to be updated automatically and directly after the control-plane update (subject to constraints
-like `maxUnavailable` in each MachineConfigPool).
+### Resources Supporting Change Management
 
-Clearly, if this was the only mode of updating worker-nodes, we could never successfully disentangle the
-concepts of control-plane vs worker-node updates in Standalone environments since one implies the other.
+Resources which support a reference to a `ChangeManagementPolicy` are said to support change management.
+Resources which support change management will implement a `spec.changeManagement` stanza. These stanzas
+must support AT LEAST the following fields:
 
-In short (details will follow in the implementation section), the assisted strategy allows users to 
-constrain which rendered machineconfig [`desiredConfig` the MachineConfigPool](https://github.com/openshift/machine-config-operator/blob/0f53196f6480481d3f5a04e217a143a56d4db79e/docs/MachineConfig.md#final-rendered-machineconfig-object) 
-should be advertised to the MachineConfigDaemon on nodes it is associated with. Like the `MaintenanceSchedule` 
-strategy, it also respects the `pausedUntil` field.
+#### ChangeManagement Stanzas
+```yaml
+kind: ClusterVersion|MachineConfigPool|HostedCluster|NodePool
+spec:
+  changeManagement:
+    # If set to "true" or a future date (represented as string), the effective change 
+    # management strategy is Disabled and the window is permissive. 
+    # Date must be RFC3339.
+    # When disabled with this field, all other values of changeManagement are ignored.
+    # This field is modeled on HCP's existing HostedCluster.spec.pausedUntil which uses a string. 
+    disabledUntil: <datetime|bool>str
+    
+    # If set to "true" or a future date (represented as string), the strategy will be
+    # paused (not permissive) unless overridden by disabledUntil. If paused with this 
+    # setting, it overrides the "policy" stanza.
+    # Date must be RFC3339. 
+    # This field is modeled on HCP's existing HostedCluster.spec.pausedUntil which uses a string. 
+    pausedUntil: <datetime|bool>str  

```

Comment:
With `disabledUntil` as a "I want to touch this despite the lack of formal maintenance window" access hatch, I think it makes sense to have that one down on the covered resource in `changeManagement`.  But I'm less clear on why `pausedUntil`'s "despite the usual maintenance window, I have reasons to want to go slowly for a bit" would be down in `changeManagement`.  Maybe `pausedUntil` belongs up on `ChangeManagementPolicy`?  Or maybe `exclude` already covers that use case at the `ChangeManagementPolicy` level, and we don't need `pausedUntil` at all?

Reply:
I wanted `pausedUntil` to be a knob even without a `ChangeManagementPolicy` defined. It is in the resource itself because I believe:
1. It offers a simple way for users to control roll-outs without having to get into the details of creating `ChangeManagementPolicy` definitions. I want them to be able to easily toggle pausing a rollout of an MCP without the rabbit hole of understanding `ChangeManagementPolicy` - that rabbit hole is only worth it for someone who wants a real `MaintenanceSchedule`.
2. Moving `pausedUntil` into `ChangeManagementPolicy ` makes the meaning even more challenging to convey / interpret. For example, how would you interpret "pausedUntil: true" for a `Restrictive` strategy vs a `Permissive` strategy? A paused restrictive strategy sounds like it would be evaluated as permissive. 
3. It permits CLI tools to pause and unpause rollouts regardless of whether an administrator has configured a `ChangeManagementPolicy`. I think this is critical in order to implement verbs like `oc adm update worker-nodes start|stop|pause` and ensure they have MCP granularity.